### PR TITLE
update PATH for the packages installed

### PR DESCRIPTION
**Pull request by manashgoswami**
**Description**: The PATH variable updated to include the packages installed by this dependencies script.

**Motivation and Context**
- this localizes the changes introduced within the same script.

### CODE REVIEW

**Comment by snnn** [comment]
Who installed miniconda?  Why do we need it?

**Comment by manashgoswami** [comment]
> Who installed miniconda? Why do we need it?

its required for container images used in Azure ML.

**Comment by snnn** [comment]
> > Who installed miniconda? Why do we need it?
> 
> its required for container images used in Azure ML.

Then why do we also install python3 and python3-dev ? In this case, we should use the python related things from anaconda, not the system one?

File: dockerfiles/install_common_deps.sh [hunk 1]
```diff
@@ -20,6 +20,9 @@ conda install -y python=3.6 numpy
 conda clean -aqy
 rm -rf /opt/miniconda/pkgs
 
+# Add conda location to PATH
+EXPORT PATH=/opt/miniconda/bin:$PATH
```
Comment (snnn) [inline dockerfiles/install_common_deps.sh]: should it be "export" instead of "EXPORT" ?

Comment (manashgoswami) [reply dockerfiles/install_common_deps.sh]: this export statement should go in the higher level dockerfile. Will resubmit the change.

**Comment by manashgoswami** [comment]
incorrect implementation.

**Pull request closed without merging**